package.json

```diff
@@ -15,6 +15,7 @@
     "url": "https://issues.apache.org/jira/browse/CB"
   },
   "scripts": {
+    "eslint": "eslint src test",
```

Comment:
can you also get it to run on `pkg`

`"eslint src test pkg"`

Reply:
Hmm so if we add in pkg here, there may be an issue because pkg is in the .gitignore. Should I remove pkg from .gitignore so eslint can test it?

Reply:
we want to keep `pkg` in `.gitignore`. What happens if we run `eslint` on a folder that doesn't exist?

We need to figure out how to fix those errors. Grr. I can take a look with you. Manually fixing those errors is not an option. Are they all fixable with `eslint --fix`?

Reply:
If I do `npm run eslint pkg/`, it will reveal all of the pkg/ errors. I can reduce some errors using --fix (mostly indenting errors) but the rest would have to be fixed manually. I won't be able to change/fix anything, however, unless I can check in the pkg/ directory though. If I run `npm run eslint` without specifying pkg, it will just ignore it. Hmm...